tests/models/jamba/test_modeling_jamba.py

```diff
@@ -650,82 +650,112 @@ def test_new_cache_format(self, num_beams, do_sample):
 class JambaModelIntegrationTest(unittest.TestCase):
     model = None
     tokenizer = None
+    # This variable is used to determine which CUDA device are we using for our runners (A10 or T4)
+    # Depending on the hardware we get different logits / generations
+    cuda_compute_capability_major_version = None
```

Comment:
This `cuda_compute_capability_major_version` pattern is copied from other models like e.g. `gemma`